ui/src/main/resources/PhenoTips/SimilarCases.xml

```diff
@@ -485,12 +489,14 @@
 
     _displayResultContactInfo : function (r) {
       var contactType = (r.myCase || r.myGroupsCase) ? "none" : (r.access == "public" ? "open" : "restricted");
-      var icon = (contactType == 'restricted') ? this.Utils.generateIcon('contact', "Contact owner for further information") : this.Utils.generateIcon('group');
+      var icon = (contactType == 'restricted' &amp;&amp; this._data.isManager) ? this.Utils.generateIcon('contact', "Contact owner for further information") : this.Utils.generateIcon('group');
```

Comment:
This is wrong, now for somebody that's not a manager of the patient, "this belongs to one of your groups" will show up, regardless whether the patient is actually accessible or not.